samples/kotlin-kotlin2js/src/main/kotlin/org/jlleitschuh/gradle/ktlint/sample/js/main.kt

```diff
@@ -0,0 +1,14 @@
+package org.jlleitschuh.gradle.ktlint.sample.js
+
+import kotlin.browser.document
+
+val secondDiv: dynamic = document.createElement("div").apply {
```

Comment:
Forgive my lack of experience. What is the `dynamic` type in Kotlin? This is the first I've ever seen it.

Reply:
It is only js specific: https://kotlinlang.org/docs/reference/dynamic-type.html